packages/chevrotain/src/parse/exceptions_public.ts

```diff
@@ -22,7 +21,7 @@ Object.freeze(RECOGNITION_EXCEPTION_NAMES)
 // hacks to bypass no support for custom Errors in javascript/typescript
 export function isRecognitionException(error: Error) {
   // can't do instanceof on hacked custom js exceptions
-  return includes(RECOGNITION_EXCEPTION_NAMES, error.name)
+  return RECOGNITION_EXCEPTION_NAMES.indexOf(error.name) !== -1
```

Comment:
looks like there is an array.prototype.includes
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes